src/components/Main.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="fixed bottom-4 right-4 bg-neutral-800 font-sans text-white px-4 rounded-lg py-2">
+  <div class="fixed bottom-4 right-4 bg-neutral-800 font-sans text-white px-4 rounded-lg py-2 z-999">
```

Comment:
Let's use `z-[99]` here instead? That way we don't need to change the tailwind config file.

Reply:
And we can revert the changes to the tailwind config file